EIPS/eip-8888.md

```diff
@@ -0,0 +1,292 @@
+---
+eip: 8888
```

Comment:
```suggestion
eip: 7204
```
Assigning EIP number as the PR number.  Please also rename the filename.

Reply:
changed, thanks